scripts/build-json/resolve-status.js

```diff
@@ -0,0 +1,12 @@
+function packageStatus(specifications, bcd) {
+  return {
+    non_standard: specifications === "non-standard",
+    experimental:
+      bcd.__compat !== undefined && bcd.__compat.status.experimental,
+    deprecated: bcd.__compat !== undefined && bcd.__compat.status.deprecated
+  };
```

Comment:
Sorry about this, I was wrong: https://github.com/mdn/browser-compat-data/pull/5816#issuecomment-598159285

Maybe something like this?

```suggestion
  const hasStatus = bcd.__compat !== undefined && bcd.__compat.status !== undefined;
  return {
    non_standard: specifications === "non-standard",
    experimental: hasStatus && bcd.__compat.status.experimental,
    deprecated: hasStatus && bcd.__compat.status.deprecated
  };
```

Reply:
Sure. I don't expect webextensions will make it into Stumptown, but this is fine too.